### DISCUSSION TITLE
include main site style in editor

### DISCUSSION THREAD

**Discussion by dchubad** [tinymce/tinymce#8404] [General]
How can I include my site .css files in the editor? When I create for instance table layouts and preview the output it does not include my site styling. My main site uses Responsee styling but none of it shows in the editor. Hope I am explaing this properly. thanks

**Comment by lostkeys** [comment]
Hi @dchubad! The [documentation on content CSS](https://www.tiny.cloud/docs/tinymce/6/add-css-options/) should be helpful.

**Reply by dchubad** [reply]
perfect, thanks. I spent hours today trying to get that working lol now it works wonderfully. much appreciated

**Reply by dchubad** [reply]
can it add more than one css file?

**Reply by TheSpyder** [reply]
Yes, it can take an array of URL strings, or you can use `@import` to include extra files from inside the one configured

**Reply by dchubad** [reply]
> Yes, it can take an array of URL strings, or you can use `@import` to include extra files from inside the one configured

thanks for this, greatly appreciated
